src/AspNet.Security.OAuth.FourSquare/FourSquareAuthenticationHelper.cs

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using Microsoft.Framework.Internal;
+using Newtonsoft.Json.Linq;
+
+namespace AspNet.Security.OAuth.FourSquare {
+    /// <summary>
+    /// Contains static methods that allow to extract user's information from a <see cref="JObject"/>
+    /// instance retrieved from FourSquare after a successful authentication process.
+    /// </summary>
+    public static class FourSquareAuthenticationHelper {
+        /// <summary>
+        /// Gets the identifier corresponding to the authenticated user.
+        /// </summary>
+        public static string GetIdentifier([NotNull] JObject user) => user.Value<string>("id");
+
+        /// <summary>
+        /// Gets the last name corresponding to the authenticated user.
+        /// </summary>
+        /// <param name="user"></param>
+        /// <returns></returns>
+        public static string GetLastName([NotNull] JObject user) => user.Value<string>("lastName");
+
+        /// <summary>
+        /// Gets the first name corresponding to the authenticated user.
+        /// </summary>
+        /// <param name="user"></param>
+        /// <returns></returns>
+        public static string GetFirstName([NotNull] JObject user) => user.Value<string>("firstName");
+
+        /// <summary>
+        /// Gets the username corresponding to the authenticated user.
+        /// </summary>
+        /// <param name="user"></param>
+        /// <returns></returns>
+        public static string GetUserName([NotNull] JObject user) =>  string.Format("{0} {1}",user.Value<string>("firstName"), user.Value<string>("lastName"));
```

Comment:
I love `string.Format` (and the new string interpolation feature is just amazing: http://www.thomaslevesque.com/2015/02/24/customizing-string-interpolation-in-c-6/), but the simplest overload uses the current culture, which is not always what you want. And here, the format was simple enough to just use a concatenation.

That said, if you prefer using `string.Format` with `CultureInfo.InvariantCulture`, I'm fine too :+1: